src/Tasks.UnitTests/AssignCulture_Tests.cs

```diff
@@ -192,6 +192,32 @@ public void Regress283991()
             Assert.Equal(0, t.AssignedFilesWithCulture.Length);
             Assert.Equal(1, t.AssignedFilesWithNoCulture.Length);
         }
+
+        /*
+        * Method:   PseudoLocalization
```

Comment:
Nit: can you make this a doc comment like the others in this file?

Reply:
On inspection of the file, they're all `/* */` format except the one immediately before this new test. Happy to update to `///` format if you prefer though.

Reply:
No, consistency with the majority is the right call. Naturally the one I looked at was the oddball 🤷‍♂️